.flake8

```diff
@@ -4,6 +4,7 @@ exclude =
     versioneer.py,
     _version.py
     src/prefect/contrib/*
+    src/prefect/tasks/*
```

Comment:
Before flake8 was passing fine on this, now it's not. Can you fix things so that it passes again?

Reply:
Flake8 appears to be passing fine, it's codecov which is failing now

Reply:
Sorry, I mean can you drop the changes to the config so that the task library is still linted by flake8? Only the new code you moved over hear should be affecting this, things were passing fine before.